src/Message/PurchaseRequest.php

```diff
@@ -70,11 +70,14 @@ private function sign($params, $api_key) {
 
 	public function sendData($data)
     {
+		// FIXME: You're not actually sending any data here.
```

Comment:
Yes, the response will be the a message with the appropriate methods to indicate that a redirect is needed and what kind of redirect it is. Your application still needs to be perform the actual redirect, though there are helper methods in the OmniPay abstract response to help you.